docker/ci/dockerfiles/build.centos7.opensearch.x64.arm64.dockerfile

```diff
@@ -114,7 +114,7 @@ CMD ["/bin/bash", "-l"]
 
 # Install ruby / rpm / fpm related dependencies
 RUN . /etc/profile.d/rvm.sh && rvm install 2.4.0 && rvm --default use 2.4.0 && \
-    yum install -y rpm-build && \
+    yum install -y rpm-build createrepo && \
```

Comment:
Can you add `yum clean all` after createrepo?
Thanks.

Reply:
updated